src/lucky/params.cr

```diff
@@ -367,8 +367,9 @@ class Lucky::Params
     multipart_params = {} of String => String
     multipart_files = {} of String => Lucky::UploadedFile
     body_io = IO::Memory.new(body)
-    boundary =
-      HTTP::Multipart.parse_boundary(request.headers["Content-Type"]).to_s
+
+    boundary = MIME::Multipart.parse_boundary(request.headers["Content-Type"]).to_s
```

Comment:
Does this need to be required? `require "mime"`

Reply:
It could. `MIME` ended up been available because it is required in the `StaticFileHandler` of the stdlib.

Reply:
hmm interesting. Yeah, the `Lucky::StaticFileHandlers` inherits from the `HTTP::StaticFileHandler` which does require mime already. Not sure why the CI said mime is undefined...